### ISSUE TITLE
Autocomplete link markdown format

### ISSUE THREAD

**Issue by avucic** [artempyanykh/marksman#83]
Is it possible to to configure autocompletion markdown link format `[Foo Bar](foo-bar.md)` instead wiki of format `[[foo-bar]]`?

**Comment by artempyanykh** [comment]
Hi @avucic! Thanks for raising this. Could you give a more specific example for the behavior you want: document + input to autocomplete? The way things are now:
1. If you type `[[` this will bring *wiki-style* autocomplete.
2. If you type `[](` or `[Foo bar](` this will bring markdown-style autocomplete. You need to input the square brackets `[]` because markdown inline links have the structure `[title](url)`.

I'm assuming you want to streamline p.2 but I'm not sure what exactly you have in mind?

**Comment by avucic** [comment]
Hi @artempyanykh. Right now I'm using [zk-nvim](https://github.com/mickael-menu/zk-nvim) which has implemented LSP sever (I beleive `zk` tool has that under the hood). I like feature that when you type `[[ node anme]]` AC will return link like this: `[h1 header from the file](path to the file.md)`. Actually you can configure that in .zk/config.toml `link-format = "[{{title}}](/{{rel-path}})"`. Currently I have to enter the link title manually. Not big deal, of course, but it would be nice if Marksman can do that for me ;) Btw. awesome work!!

**Comment by MilanVasko** [comment]
Indeed, a configuration option such as the one @avucic mentioned would also solve my use case really nicely :+1:
